Redesigned and improved the Notifications page

MMDJafari opened the pull request:
Redesigned and improved the Notifications Page & some minor fixes
https://github.com/MMDJafari/freefeed-react-client/tree/notification-page
-------------------------------------------------------------------------------------------------------------
Update: Added Inline Video Player & some minor fixes
https://github.com/MMDJafari/freefeed-react-client/tree/notification-page-%26-inline-video-player
-------------------------------------------------------------------------------------------------------------
Update: Moved the Navigation Bar to the bottom of the screen for mobile devices & some minor fixes
https://github.com/MMDJafari/freefeed-react-client/tree/notification-page-%26-inline-video-player-%26-navigation-bar
-------------------------------------------------------------------------------------------------------------
Live Test: https://freefeed.vercel.app
-------------------------------------------------------------------------------------------------------------
Screenshots:

Mobile:
![Screenshot_20220906-014127_Chrome](https://user-images.githubusercontent.com/66924476/188513722-c7395d08-cd97-42d0-849e-e1ef2e284ede.png)

Desktop:
![Screenshot from 2022-09-06 01-36-12](https://user-images.githubusercontent.com/66924476/188513769-fd20f6dc-ae0b-4596-847e-9f0a00a1b294.png)

MMDJafari committed:
Redesigned and improved Notification page

MMDJafari committed:
Update notification-body.jsx

MMDJafari committed:
Redesigned and improved Notification page

MMDJafari committed:
fix mouse hover

MMDJafari committed:
fix mouse hover

MMDJafari committed:
Update notification-body.jsx

MMDJafari committed:
fix user tag remover

davidmz commented on config/default.js:
```diff
@@ -6,7 +6,7 @@ const DAY_IN_MILLISECONDS = 1000 * 60 * 60 * 24;
 export default {
   api: {
-    root: 'https://candy.freefeed.net',
+    root: 'https://freefeed.net',
```
You should not change the default settings, they purposely specify a stage instance (candy).

For testing purposes, you can override them in the config.json file at the root of the repository. It will not be included in the commit by .gitignore.

MMDJafari committed:
revert back to shows mentioned users in text and parse it with peaceoftext

MMDJafari committed:
yarn lint

MMDJafari committed:
remove clock icon

MMDJafari committed:
improve spacing between lines

MMDJafari committed:
fix showing text & profile picture for mobile devices

MMDJafari committed:
improve spacing between lines

MMDJafari committed:
fix url for vercel

MMDJafari committed:
add IG reel preview

MMDJafari committed:
revert to default config

MMDJafari commented:
Yeah, you're right but I have a problem with Vercel deployment that needs
this (and other) configs to work properly due to connecting to my repo
directly. For now, I solved it by creating another branch till I find a
better solution.

On Thu, Aug 25, 2022, 10:27 PM David Mzareulyan ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In config/default.js
> <https://github.com/FreeFeed/freefeed-react-client/pull/1548#discussion_r955266705>
> :
>
> > @@ -6,7 +6,7 @@ const DAY_IN_MILLISECONDS = 1000 * 60 * 60 * 24;
>
>  export default {
>    api: {
> -    root: 'https://candy.freefeed.net',
> +    root: 'https://freefeed.net',
>
> You should not change the default settings, they purposely specify a stage
> instance (candy).
>
> For testing purposes, you can override them in the config.json file at the
> root of the repository. It will not be included in the commit by .gitignore.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/FreeFeed/freefeed-react-client/pull/1548#pullrequestreview-1085880801>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AP6S7PDACMFWWUS5SK7VYNTV26X2FANCNFSM57QZLIFQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

MMDJafari committed:
Play video attachments using inline video tag - using @n1313 fork

MMDJafari committed:
Revert "Play video attachments using inline video tag - using @n1313 fork"

This reverts commit c846debf700d9a1b137270a8e701d5277b6914ed.

MMDJafari committed:
Play video attachments using inline video tag - using @n1313 fork

MMDJafari committed:
add .webm & .ogg to inline video player

MMDJafari committed:
stick navigation bar to the bottom of screen

stick navigation bar to the bottom of screen for mobile devices with new icons and fix both them for navigation bar

MMDJafari committed:
add dark subscription request alert style

MMDJafari committed:
rebase everything
